### ISSUE TITLE
TWE: Create Template Issue: Create Q&A in Easy Retro for interviews

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#361]
### Dependency
- [x] #362 will help us evaluate how EasyRetro has performed and offer best practice when using EasyRetro

### Overview
We need to create a template issue for `TWE: Research: Easy Retro boards: Create` so that we have a consistent, transparent, defined, and self-supporting structure for people creating them. 

#### Customize this issue
 - Customize this issue by adding the Name of Issue Template. 
    - _Note: An issue template's name is different from the issue title created from the template._
    - [x] Search for `[Replace NAME OF ISSUE TEMPLATE]` by using the browser search (**Mac**: ⌘+F; **PC**: Ctrl+F)
    - [x] Replace the keyword string (including text and the brackets) below with the name of your issue template according to the template naming convention: `TWE: Role: Direct object: Action verb`
        - The role is the primary owner of the issue template (e.g., Research, Design, Product)
        - Direct object should be the item that we're updating or working on (e.g., Wiki, Interviews, Roadmap)
        - Action verb should be the main descriptor of the action items (e.g., Update, Create, Transcribe)

#### Customize the issue settings (located in right sidebar)
Change the issue settings to ensure the issue is properly categorized and easy to manage.
- [x] Under Assignees, assign yourself.
- [x] Under Projects, choose gear, choose Repository, and choose TWE: Project Board.
- [x] Add the milestone: Project Management
- [x] Add a feature label specific to what the template is related to, if available.  If not clear what you should use, ask product.

#### Customize template draft
Create a draft of new template, which will be known as our "draft".
- [x] Copy resource 1.01 Content to copy for draft to a new comment on this issue.
- [x] In your draft, fill out the top part (called front matter) with the following information.  See resource 1.02 Front Matter Resources section below for definitions and examples.
    - [x] Name
    - [x] About
    - [x] Title
    - [x] Labels
- [x] Define any Dependencies (or remove the dependency section if there is no dependency)
- [x] Customize the Overview section
- [ ] Fill out the Action Items section (in your draft) by defining the steps that are required to be done, keeping the following in mind:
   - [ ] The person working on this issue might be new to the org, so the issue should be able to be worked on without any prior knowledge
   - [ ] If the person needs to add links, or put documents in specific locations, the place where they can do that action must be linked in the resources section
      - [ ] The links in the resources section provided by the issue creator should be in the same order as the Action Items 
      - [ ] Refer to resource 1.03 Example Action Items for Finding Wiki Page and Template for an example of how to write clear instructions that avoid compound steps (steps within a step).
   - [ ] Number each resource in the order each resource appears in the Action Items
       - [ ] Group resources into numbers and decimals (e.g., Resource 1.01, Resource 2.01)
       - [ ] Group the resources based on whether they are available when making the issues or they need to be created from this issue
           - If the resource is available, then group it under the **Resources for creating this issue** section (e.g., Resource 1.01, Resource 1.02, Resource 1.03)
           - If the resource will be delivered or created from this issue, then group it under the **Resource links gathered during the making of this issue** section (e.g., Resource 2.01, Resource 2.02, Resource 2.03)
- [ ] If documents or folders (sheets, docs, presentations, dashboard, etc.) will be created, find the document template that should be used by checking resource 1.04 WIKI Templates by role pages.  
  - [ ] If there is a template
     - [ ] Add a link to the template into the ` #### Resources for creating this issue`  section of your draft
  - [ ] If there is no template yet
    - [ ] Put a placeholder into the` #### Resources for creating this issue`  section of the draft `[TWE: Template Issue: [Role]: [Replace ACTION VERB][Replace DIRECT OBJECT]`
- [ ] Get sign-off by Product on the draft

#### If this is for a template draft that needs testing at a later stage, please use the following handoff steps:
- [ ] Add `feature: template creation (step 1)` to this issue
- [ ] Once the template draft is finished, assign a team member or yourself to create testing issues 
- [ ] Once the testings issues are completed, get sign-off by Product on the template draft
- [ ] Create or update the template issue on the repo 
- [ ] Add the link to the new issue template once created to resource 2.01.  Find by clicking on the **Get started** button to the right of the new template listed on [new issue page](https://github.com/hackforla/internship/issues/new/choose)
- [ ] Update the appropriate place in the wiki for your template (see resources section)

#### Resources for creating this issue
- 1.01 Content to copy for draft
```
Draft of template issue for `[Replace NAME OF ISSUE TEMPLATE]`
---
**name:** TWE: Template Issue: [Replace ROLE]: [Replace ACTION VERB][Replace DIRECT OBJECT]
**about:** This issue helps you structure your template issue to be consistent with our other template issues in [Replace DIRECT OBJECT]
** title:** TWE: [Replace ROLE]: [Replace DIRECT OBJECT]: [Replace ACTION VERB]
**labels:** 'feature: board / google drive / tools, feature: missing, milestone: missing, role: missing, size: missing', 'Participant Type: missing', 'complexity: missing'
**assignees:**

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Copy and paste entire template text in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
    - [ ] Search for `[Replace`
    - [ ] Replace with whatever the keyword or information is asking for.
- Customize the issue settings (located in right sidebar)
    - [ ] Under Labels, add labels for any labels identified as missing
    - [ ] Remove the corresponding missing labels
    - [ ] Under Projects, choose gear, choose "Repository," and choose TWE: Project Board.
    - [ ] Add the milestone: [Replace MILESTONE]
    - [ ] Add a feature label specific to what the template is related to, if available. If not clear what you should use, ask product.
- Customize dependencies 
    - [ ] If no dependencies, delete dependency section
    - [ ] If dependencies, add the dependencies (any issue or reason that would prevent the action items from being completed)
- Remove this section.
    - [ ] Delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
We need to [Replace ACTION VERB] [Replace DIRECT OBJECT] for [Replace REASON].

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, coding, etc.

### Resources
#### Resources for creating this issue
- 1.01
- 1.02

#### Resources gathered during the completion of this issue
- 2.01
- 2.02
```

- 1.02 Front Matter Resources
   <details>
   <summary>Front Matter Fields and Meaning</summary>

   - name: (name that appear on the new issue tab)
   - about: (is the description that appears next to the issue template on the new issue tab)
   - title: (the title that appears when the issue is created.  User then customizes this title for their issue)
       - Use the naming convention: TWE: Role: Direct object: Action verb
       - If more than one word should go into the direct object spot, use multiple square brackets with the keyword (see example below)
   - labels: (choose the labels that will be the same each time, and choose missing labels for the ones that depend on content)
   - assignees:  
   </details>
   <details>
   <summary>Front Matter Field Example</summary>

   - name: 'TWE: Template Issue: Research: Create Roadmap'
   - about:  This issue helps you structure your template issue to be consistent with our other template issues in roadmap
   - title: 'TWE: Research: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF RESEARCH METHOD]: Roadmap : Create'
   - labels: 'feature: research', 'feature: roadmap', 'milestone: missing', 'role: UI/UX research', 'Participant Type: missing', 'Research 000: missing', 'complexity: missing'
   - 'size: 0.50pt'
   - assignees: ''
   </details>

- 1.03 Example Action Items for Finding Wiki Page and Template
   <details>
   <summary>Example Action Items</summary>

   ```
   ### Action Items
   - Customize wiki page
        - [ ] Go to link 1.01 Internship Wiki, Join the team page
        - [ ] Find wiki page for this issue by looking at role pages.
        - [ ] Copy the link for the wiki page.
        - [ ] Add the link to Resource 2.01
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   - Customize document template.
       - [ ] Go to link 1.02 Design Process Overview
       - [ ] Find document template that should be used in this issue.
       - [ ] Copy the link for the template.
       - [ ] Add the link to Resource 2.02
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   ```

   </details>

   <details>
   <summary>Example Resource Lists</summary>

   ```
   ### Resources for creating this issue
   - 1.01. [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team)
   - 1.02. [Design Process Overview](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

   ### Resource links gathered during the making of this issue
   - 2.01. [Wiki page for [Replace TYPE OF ROLE]]
   - 2.02. [TWE: Template: [Replace NAME OF TEMPLATE]]
   ```

   </details>

- 1.04 WIKI Templates by role pages
   - [TWE: WIKI: UX Research Guides Templates](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates) 
   - [Design](https://github.com/hackforla/internship/wiki/Design-Process-Overview)
   - [Product](https://github.com/hackforla/internship/wiki/Product)
   - [Business Analyst / Data Science]
   - [Software Engineer]
   - [Marketing]
   - default document templates, in case you don't have one yet, can be found on the [Design Process Overview page](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

#### Resources gathered during the completion of this issue
- 2.01 [TWE: [Replace ROLE]: [Replace DIRECT OBJECT]: [Replace ACTION VERB]]([Replace LINK])

**Comment by pandanista** [comment]
Draft of [replace with name of type] template issue
---
name: TWE: Create Template Issue: [replace with name of template]
about: 
title: TWE: [replace with action verb] [replace with name of template]
labels: 'feature: board / google drive / tools', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'
assignees: 
---
# Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- [ ] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [ ] Find document template that should be used [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link into the resources section
- [ ] If no dependencies, delete dependency section
- [ ] any place you see `[replace...]` replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] add appropriate labels and remove the corresponding missing labels
- [ ] delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

### Resources

**Comment by pandanista** [comment]
@priyatalwar Two asks here. 

1. After the #362 gets signed off by product, we can check off the dependency box at the top and remove the dependency label for this issue, and put this one into new issue approval column. At the same time, we can check off the boxes under "remove dependency" section at the end of issue #362.
2. If this issue is approved, please move it to the prioritized backlog so I can keep working on it. 

Thank you!

**Comment by pandanista** [comment]
In the template for easy retro, instead of focusing on creating new Easy Retro boards, we should just provide [a consolidated, commonly asked questions and answers document](https://docs.google.com/document/d/10tvWtCCIaxGisTUgGq4So4aQ5h7xhUNAz0E54pEjaM0/edit?usp=sharing) so team members can customize the Q&As based on the research plan and goals.

The template should cover both setting up Easy Retro board and a list of consolidated Q&As, and instruct team members to clean up and consolidate post interview sessions for every RP. Will update the template (maybe need to split them into two templates).

**Comment by pandanista** [comment]
- https://github.com/hackforla/internship/issues/361#issuecomment-1430710942 is a rough first draft, and it only has instructions on how to set up EasyRetro boards.
- https://github.com/hackforla/internship/issues/361#issuecomment-2384140710 has more context on what we need for the template. We might need to have two templates based on this comment.
  - Template A is to consolidate all the Q&A from existing boards, and #362 was the issue we tested it out and can be worked into a template for consolidating Q&A; 
  - Template B is to set up an EasyRetro board, like in the comment, https://github.com/hackforla/internship/issues/361#issuecomment-1430710942. 
    - Easy Retro has updated their plan and features, so there are now more limitations. 
      - https://intercom.help/easyretro/en/articles/4892108-how-the-free-version-of-easyretro-works
      - https://intercom.help/easyretro/en/articles/6928260-i-can-t-create-a-new-public-board-in-my-dashboard-it-says-that-i-still-have-public-boards-left

**Comment by gayathri3120** [comment]
Progress: I have gone through the dependency and the current issue and the template in https://github.com/hackforla/internship/issues/361#issuecomment-1430710942
Have built out a template for this issue, still working on it
Blockers: Haven't gotten the access to easy retro

**Comment by gayathri3120** [comment]
Internship UI/UX 1Password Vault access requested

**Comment by gayathri3120** [comment]
I received the access, currently working on creating the templates 
1. to set up easy retro
2. to consolidate the questions

**Comment by gayathri3120** [comment]
Draft of template issue for `Create Q&A Boards in Easy Retro for Interviews`
---
**name:** TWE: Template Issue: Create Q&A Boards in Easy Retro for Interviews
**about:** This issue helps you structure your template issue to be consistent with our other template issues in Creating Q&A Boards in Easy Retro for Interviews
**title:** TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Easy Retro [Replace TYPE OF Board] Interview Q&A: Create
**labels:** 'feature: research', 'feature: interview(s)', 'Participant Type: missing', 'Research 000: missing', 'size: 0.5pt', 'role: UI/UX research', 'milestone: missing', 'complexity: missing'

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Open the "Research Overview" wiki page (Resource 1.01)
    - [ ] Choose the appropriate research roadmap or research plan folder (ie., RP###) on the wiki page 
       - Use the information on the wiki page and in research roadmap or research plan to customize this issue
   - [ ] Copy and paste entire template in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
   - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
        - Year 
            - [ ] Find/Replace `[Replace YY]` with cohort/research year (Two digit format, ex: "22" for year "2022") 
        - Research Plan Number
             - [ ] Find/Replace `[Replace 000]` with the research plan number (Three digit format, ex: "001" for research plan "RP001") 
        - Type of Participant
            - [ ] Find/Replace `[Replace TYPE OF PARTICIPANT]` with the participant type (ex: "Intern" or "Mentor")  
        - Type of Interview
            - [ ] Find/Replace `[Replace TYPE OF INTERVIEW]` with the type of interview (ex: "Intake" or "Exit")
            - [ ] Find/Replace `[Replace TYPE OF BOARD]` with the type of Easy Retro board that is being created as communicated by the UX Lead (ex: "Training," "Practice,", or "Real-time") 
    - [ ] Copy and paste the edited template from text editor back into this Github Issue, replacing all original text
    - [ ] Re-check all the checkboxes (all of the `[Replace...]` should be replaced by the correct information)
    - [ ] Close the text editor
    - Update the issue title
        - [ ] Copy the following
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Easy Retro [Replace TYPE OF Board] Interview Q&A: Create
        ```
        - [ ] Choose "Edit" in the title section of this issue
        - [ ] Paste the text you copied to replace the issue title
        - [ ] Choose "Save"

- Customize the issue settings (located in right sidebar)
    - [ ] Under Assignees, assign a Researcher if it has been determined. Otherwise, leave it blank.
    - [ ] Under Projects, choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), choose `Repository`, and choose `Program Management`
    - [ ] Add the milestone: choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and choose `Research Plan Execution`
    - [ ] Under Labels, remove the `milestone: missing` label
- [ ] Under Labels,  choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Research: RP[Replace 000]`
    - [ ] Under Labels, choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Participant Type: [Replace TYPE OF PARTICIPANT]`    
    - [ ] Under Labels: remove the `Research 000: missing` and `Participant Type: missing`
- Remove this section
    - [ ] After all the checkboxes above are complete and the issue is ready to be worked on, delete all text below the "Instructions for creating this issue" (top heading) and above "Dependencies" (heading below) -- i.e., the issue assignee will never see this part

### Dependencies

### Overview
We need to Create Q&A Boards in Easy Retro for RP[Replace 000] so we can provide answers to team members' questions to interview instructions as well as questions during practice sessions and real time interviews.

### Action Items
- [ ] UX lead asks admin to add the team member working on this issue to the 1Password internship-UI/UX's vault
- [ ] Confirm with the assignee that they have access to the "TWE/Internship Shared Google Drive"
#### Customize Resource Links
- Customize Resource for Wiki Page Link
    - [ ] Go to the wiki page: Research Output Overview (Resources # 1.01)
    - [ ] Choose the link in the Research by Plan Number section
    - [ ] Find relevant wiki page for RP[Replace 000]
    - [ ] Copy the link for the RP[Replace 000] wiki page.
    - [ ] Update Resource # 2.01 with the link you just copied. Place it into parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Save" in Github and make sure all the checkboxes above have been checked
- Customize Resource for this Research Plan's Google Drive Folder
    - [ ] Open the Google Drive's Research by Type Folder (Resources  # 1.02)
    - [ ] Open the `[Replace TYPE OF PARTICIPANT]` Folder
    - [ ] Copy the link for the `RP[Replace 000]` folder
        1. Right-click on the `RP[Replace 000]` folder
        1. Choose "Share"
        1. Choose "Copy link"
    - [ ] Update Resource # 2.02 with the link you just copied. Paste into parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink.
    - [ ] Choose "Save" in Github and make sure all the checkboxes above have been checked
####  Create Easy Retro for RP[Replace 000] [Replace TYPE OF Board] Interview
- [ ] Log into Easy Retro using 1Password 
    1. Log into 1password (Resources # 1.03)
    2. Choose P:TWE - UI/UX (Legacy) vault
    3. Choose EasyRetro for [Replace TYPE OF Board] Interviews to see login info for Easy Retro account
    4. Go to Easy Retro (Resources # 1.04)
    5. Log into Easy Retro with the email account using the login information in 1Password
- [ ] Create an Easy Retro for Q&A so we can address Interviewer's questions regarding the interview instructions
    1. Choose "Add board"  <img width="100" alt="Screenshot 2023-02-14 at 8 49 45 PM" src="https://user-images.githubusercontent.com/57029070/218937523-1ebba84d-05f5-4270-98df-4f3ab0599d7d.png">
    1. Copy     
    ```
    TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] [Replace TYPE OF Board] Interview Q&A
    ```
    3. Paste into the top field to rename the board
    4. Make sure the board is in column format  <img width="40" alt="Screenshot 2023-02-14 at 8 56 21 PM" src="https://user-images.githubusercontent.com/57029070/218937761-ef0b4637-aceb-45e4-8f6b-a95572f3c0fd.png">
    5. Create two columns, `Questions` and `Answers` (see example screenshot below)   <img width="700" alt="Screenshot 2023-02-14 at 8 50 47 PM" src="https://user-images.githubusercontent.com/57029070/218937587-837efdb9-686b-4987-9d5e-e9f030750d29.png">
    6. Choose the 3 vertical dots, and choose "Set column color" to ensure the two columns are in different colors  <img width="100" alt="Screenshot 2023-02-14 at 8 59 56 PM" src="https://user-images.githubusercontent.com/57029070/218937867-466f16d2-084c-4c56-aa46-d338e66c25d2.png">
- [ ] Choose "Share"  <img width="40" alt="Screenshot 2023-02-14 at 9 02 36 PM" src="https://user-images.githubusercontent.com/57029070/218937964-e0226687-f6f8-47d9-87b6-71b4d22b25e8.png">
- [ ] Choose "Copy URL"
- [ ] Update Resources # 2.03 by pasting the copied link inside parentheses at the end of the line
- [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
- [ ] Review with UX Lead
- [ ] Product sign-off
    

### Resources/Instructions
#### Resources for creating this issue
1.01 [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1.02 [Google Drive Folder: Research by Type](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=share_link)
1.03 [1password](https://start.1password.com/signin?l=en)
1.04 [Easy Retro](https://easyretro.io/)

#### Resources gathered during the completion of this issue
2.01 [Wiki: Research Plan [Replace 000]] 
2.02 [Google Drive Folder: RP[Replace 000]]
2.03 [[TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] [Replace TYPE OF Board] Interview Q&A]

**Comment by gayathri3120** [comment]
Draft of template issue for `Consolidate Q&A Boards in Easy Retro for future Interviews`
---
**name:** TWE: Template Issue: Consolidate Q&A Boards in Easy Retro for future Interviews
**about:** This issue is used to consolidate all the questions from a Research Plan into the Consolidated Q&A Document
**title:** TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Easy Retro [Replace TYPE OF Board] Interview Q&A: Consolidate
**labels:** 'feature: research', 'feature: interview(s)', 'Participant Type: missing', 'Research 000: missing', 'size: 0.5pt', 'role: UI/UX research', 'milestone: missing', 'complexity: missing'

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Open the "Research Overview" wiki page (Resource 1.01)
    - [ ] Choose the link in the Research by Plan Number section
    - [ ] Choose the appropriate research roadmap or research plan folder (ie., RP###) on the wiki page 
       - Use the information on the wiki page and in research roadmap or research plan to customize this issue
   - [ ] Copy and paste entire template in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
   - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
        - Year 
            - [ ] Find/Replace `[Replace YY]` with cohort/research year (Two digit format, ex: "22" for year "2022") 
        - Research Plan Number
             - [ ] Find/Replace `[Replace 000]` with the research plan number (Three digit format, ex: "001" for research plan "RP001") 
        - Type of Participant
            - [ ] Find/Replace `[Replace TYPE OF PARTICIPANT]` with the participant type (ex: "Intern" or "Mentor")  
        - Type of Interview
            - [ ] Find/Replace `[Replace TYPE OF INTERVIEW]` with the type of interview (ex: "Intake" or "Exit")
            - [ ] Find/Replace `[Replace TYPE OF BOARD]` with the type of Easy Retro board that is being consolidated as communicated by the UX Lead (ex: "Training," "Practice,", or "Real-time") 
    - [ ] Copy and paste the edited template from text editor back into this Github Issue, replacing all original text
    - [ ] Re-check all the checkboxes (all of the `[Replace...]` should be replaced by the correct information)
    - [ ] Close the text editor
    - Update the issue title
        - [ ] Copy the following
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Easy Retro [Replace TYPE OF Board] Interview Q&A: Consolidate
        ```
        - [ ] Choose "Edit" in the title section of this issue
        - [ ] Paste the text you copied to replace the issue title
        - [ ] Choose "Save"

- Customize the issue settings (located in right sidebar)
    - [ ] Under Assignees, assign a Researcher if it has been determined. Otherwise, leave it blank.
    - [ ] Under Projects, choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), choose `Repository`, and choose `P: TWE: Project Board`
    - [ ] Add the milestone: choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and choose `Research Plan Execution`
    - [ ] Under Labels, remove the `milestone: missing` label
- [ ] Under Labels,  choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Research: RP[Replace 000]`
    - [ ] Under Labels, choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Participant Type: [Replace TYPE OF PARTICIPANT]`    
    - [ ] Under Labels: remove the `Research 000: missing` and `Participant Type: missing`
- Remove this section
    - [ ] After all the checkboxes above are complete and the issue is ready to be worked on, delete all text below the "Instructions for creating this issue" (top heading) and above "Dependencies" (heading below) -- i.e., the issue assignee will never see this part

### Overview
We need to consolidate all the Q&As collected from the RP[Replace 000] EasyRetro boards so we can reference those Q&As for future interview trainings and interview sessions.

### Action Items
- [ ] UX Lead asks admin to add the team member working on this issue to the 1Password's Intership-UI/UX vault 
- [ ] Confirm with the assignee that they have access to the "TWE/Internship Shared Google Drive"

#### Locate all the EasyRetro boards from previous interviews
- [ ] In Shared Drive (Resources 1.03), conduct 2 searches using `EasyRetro` and `Easy Retro` respectively
- [ ] Go through the docs and decks where `EasyRetro` or `Easy Retro` were mentioned
- [ ] Identify the existing EasyRetro Q&A boards for RP[Replace 000].
- [ ] List each board's name under the subsection of Resources 2.01
- [ ] Update the URL to each of the existing EasyRetro Q&A board inside parentheses at the end of the line in the subsection of Resources 2.01(eg: 2.01.01, 2.01.02 etc.)
- [ ] Log into EasyRetro using 1Password to double check the existing boards 
    1. Log into 1password (Resources # 1.04)
    2. Choose internship-UI/UX vault
    4. Choose EasyRetro for [Replace TYPE OF Board] Interviews to see login info for Easy Retro account
    4. Go to Easy Retro (Resources # 1.05)
    5. Log into Easy Retro with the email account using the login information in 1Password
    7. Double check if there is any board missing from the Resources 2.01 section. If so, add them to the 2.01 subsection.     
#### Customize the Google Doc structure
- [ ] Go to (Resources # 1.02)
- [ ] Copy `RP006: Interview Instructions Q&A` and `Q: What is Zoom?` on the third page of the Google Doc <img width="854" height="192" alt="Image" src="https://github.com/user-attachments/assets/39c5ad86-c98b-4f71-83e4-2a06cb0fa8e1" />
- [ ] Paste the texts you copied into a new page at the end of the document.
- [ ] Copy
```
RP[Replace 000]: [Replace TYPE OF BOARD] Interview Q&A
```
- [ ] Paste the texts you just copied in place of the `RP006: Interview Instructions Q&A` that you copied into the new page.
- [ ] Copy the questions and answers from RP[Replace 000] [Replace TYPE OF BOARD] Interview EasyRetro boards.
- [ ] Paste them in place of the `Q: What is Zoom?` of the new section you created in the Google Doc one by one
- [ ] Read the Q&A you just copied and pasted and make sure they make sense
    - [ ] Remove redundant questions
    - [ ] Make edits to the questions and answers when they are not clear enough
    - [ ] If there are any PII, flag them and talk to the UX Lead in order to remove them
- [ ] Go to the second page of the Google Doc and click on the content part of the Table of Contents. Click on the <img width="56" height="56" alt="Image" src="https://github.com/user-attachments/assets/e7ade52b-1980-4a9f-87ed-84ef9596459b" /> to refresh the Table of Contents.
<img width="1470" height="276" alt="Image" src="https://github.com/user-attachments/assets/3299ebe6-6395-4410-a471-0b8f61e0a28c" />

- [ ] Review with UX Lead
- [ ] Product sign-off

#### Deleting the Existing Easy Retro Q&A Boards to make space for new ones
- [ ] Login to Easy Retro using (Resources # 1.05) and the login information from last step, then go to the Easy Retro Dashboard.
- [ ] Open the Q&A Board/Boards from which you copied and pasted all the questions and answers in the "Locate all the EasyRetro boards from previous interviews" step
- [ ] Read the Q&A and make sure all of them are copied to the Consolidated Google Doc.
- [ ] Click on the Settings button in the Top Right corner <img width="102" height="40" alt="Image" src="https://github.com/user-attachments/assets/72592be8-2f5a-4daa-bfd3-e267627aad3f" />
- [ ] A Settings pane opens on the Right side. Scroll to the bottom of the Settings pane and select `Delete board` from the `Danger Zone`<img width="162" height="188" alt="Image" src="https://github.com/user-attachments/assets/ab715cdc-bd3c-42c4-a4b4-04974861362e" />
- [ ] A message shows up saying `Are you sure you want to delete the board?`. Click the checkbox below and click `Yes, delete it!` <img width="438" height="320" alt="Image" src="https://github.com/user-attachments/assets/9a00c820-47a5-4e12-acde-3a1ef3fda391" />
- [ ] Continue this process and delete all the Easy Retro Q&A Boards for RP[Replace 000] [Replace TYPE OF Board] Interviews(if there are multiple), after making sure that you copied all the questions and answers into the Consolidated Google Doc.
- [ ] After deleting, comment a screenshot of the Easy Retro Dashboard by taking a screenshot, copying the image and pasting it in a comment.


### Resources/Instructions
#### Resources for creating this issue
1.01 [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1.02 [TWE: Consolidated EasyRetro Q&A](https://docs.google.com/document/d/10tvWtCCIaxGisTUgGq4So4aQ5h7xhUNAz0E54pEjaM0/edit?usp=sharing)
1.03 [TWE/Internship Shared Google Drive](https://drive.google.com/drive/folders/1z3I2SLQzYWBHIypNZO-sTdSO0A8c5pvt?usp=share_link)
1.04 [1password](https://start.1password.com/signin?l=en)
1.05 [EasyRetro](https://easyretro.io/)

#### Resources gathered during the completion of this issue
2.01 EasyRetro [Replace TYPE OF Board] Interview Q&A boards

**Comment by gayathri3120** [comment]
Progress: Completed creating the template for Consolidating the Easy Retro Q&A Boards. Still working on the Creation template. 
Questions: locating easy retro boards using Resource 1.03 is confusing. Would it be ok to ask the assignee to directly check in Easy Retro for the boards relevant to the RP they are working on?
Need to discuss whether Product sign off should be before deleting boards or after or both
If the assignee is required to submit a screenshot to show they deleted the boards, where should they be asked to submit it? In the comments?

**Comment by gayathri3120** [comment]
Notes:
Discussed Easy Retro and the template in UI/UX Research meeting on Jan 27th, 2026. 
Easy Retro free plan which is currently in use only allows 3 boards at a time whether archived or not. 
Also, only a single new board can be added every month. Which means multiple Research Plans wouldn't be able to create Easy Retro boards in the same month, they would have to wait another month if a board was already created that month.
Also, Only one public board can be published at a time, so if there is already a board in Easy Retro and a Research Plan needs to create a new board, the old one would have to be consolidated and deleted before making a new one.
Here is a link of the details and limitations of the free plan:
https://intercom.help/easyretro/en/articles/4892108-how-the-free-version-of-easyretro-works

**Comment by gayathri3120** [comment]
As of the All Team Meeting on Feb 5th, 2026:
Due to the EasyRetro limitations, we need to create 2/3 new accounts in EasyRetro for making new boards. 1 account for each session(training, practice, actual interviews) 
Already created 1 new account in EasyRetro and added to 1Password
We will have to create an Epic issue to keep track of the issues for each RP
We also need to add an action item to ask the UX Lead to check for PII in the Q&A before consolidation. 
The assignee has to take a screenshot of the board before consolidation and put it in the comments
Question: After consolidation, should the assignee delete the board or keep on creating new accounts for new RP?

**Comment by gayathri3120** [comment]
Draft of template issue for `Epic Issue for Easy Retro for Interviews`
---
**name:** TWE: Template Issue: Epic Issue for Easy Retro for Interviews
**about:** This issue helps you create, structure, consolidate and then delete Q&A Boards in Easy Retro for Interviews
**title:** TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Easy Retro Interview Q&A: Epic Issue
**labels:** 'feature: research', 'feature: interview(s)', 'Participant Type: missing', 'Research 000: missing', 'size: 0.5pt', 'role: UI/UX research', 'milestone: missing', 'complexity: missing', 'epic'

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Open the "Research Overview" wiki page (Resource 1.01)
    - [ ] Choose the appropriate research roadmap or research plan folder (ie., RP###) on the wiki page 
       - Use the information on the wiki page and in research roadmap or research plan to customize this issue
   - [ ] Copy and paste entire template in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
   - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
        - Year 
            - [ ] Find/Replace `[Replace YY]` with cohort/research year (Two digit format, ex: "22" for year "2022") 
        - Research Plan Number
             - [ ] Find/Replace `[Replace 000]` with the research plan number (Three digit format, ex: "001" for research plan "RP001") 
        - Type of Participant
            - [ ] Find/Replace `[Replace TYPE OF PARTICIPANT]` with the participant type (ex: "Intern" or "Mentor")  
        - Type of Interview
            - [ ] Find/Replace `[Replace TYPE OF INTERVIEW]` with the type of interview (ex: "Intake" or "Exit")
    - [ ] Copy and paste the edited template from text editor back into this Github Issue, replacing all original text
    - [ ] Re-check all the checkboxes (all of the `[Replace...]` should be replaced by the correct information)
    - [ ] Close the text editor
    - Update the issue title
        - [ ] Copy the following
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Easy Retro Interview Q&A: Epic
        ```
        - [ ] Choose "Edit" in the title section of this issue
        - [ ] Paste the text you copied to replace the issue title
        - [ ] Choose "Save"

- Customize the issue settings (located in right sidebar)
    - [ ] Under Assignees, assign a Researcher if it has been determined. Otherwise, leave it blank.
    - [ ] Under Projects, choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), choose `Repository`, and choose `Program Management`
    - [ ] Add the milestone: choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and choose `Research Plan Execution`
    - [ ] Under Labels, remove the `milestone: missing` label
- [ ] Under Labels,  choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Research: RP[Replace 000]`
    - [ ] Under Labels, choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Participant Type: [Replace TYPE OF PARTICIPANT]` 
    - [ ] Under Labels: remove the `Research 000: missing` and `Participant Type: missing`
- Remove this section
    - [ ] After all the checkboxes above are complete and the issue is ready to be worked on, delete all text below the "Instructions for creating this issue" (top heading) and above "Dependencies" (heading below) -- i.e., the issue assignee will never see this part

### Dependencies

### Overview
We need to Create, Consildate and then Delete Q&A Boards in Easy Retro for RP[Replace 000] so we can provide answers to team members' questions to interview instructions as well as questions during practice sessions and real time interviews. After consolidating the questions and answers into the appropriate Google document, we need to delete the Boards to make space for new boards.

### Action Items
- [ ] UX lead asks admin to add the team member working on this issue to the 1Password internship-UX's vault
- [ ] If each issue creation is assigned to different assignees, UX Lead communicates the Type of Board they will be creating for (ex: "Training," "Practice,", or "Real-time") 

- [ ] Go to Resources 1.01 and check to see if any Easy Retro Board creation or consolidation issues have been created. 
   - [ ] Create Easy Retro Q&A Board for Training Interviews 
   - [ ] Create Easy Retro Q&A Board for Practise Interviews 
   - [ ] Create Easy Retro Q&A Board for Real-time Interviews 
   - [ ] Consolidate Easy Retro Q&A Board for Training Interviews 
   - [ ] Consolidate Easy Retro Q&A Board for Practise Interviews 
   - [ ] Consolidate Easy Retro Q&A Board for Real-time Interviews 
- [ ] If the Creation or Consolidation issue for one interview has already been created,
   - [ ] Link the existing issue as a sub issue in the `Sub-issues` section of this issue.
- [ ] If the Creation or Consolidation issue for one interview has not been created,
   - [ ] Use template in Resources 1.02 to create new Create issues and the template in Resources 1.03 to create new Consolidation issues;
   - [ ] Link each new issue as a sub issue in the `Sub-issues` section of this issue.
- [ ] Link each issue under blocked by, in `Relationship` section (right side) of issue.
- [ ] In the new issue you just created, go to the section of `Instructions for creating this issue`.
- [ ] Customize each issue by following the steps in the section of `Instructions for creating this issue`, including the issue title, body, labels, project board, milestone, etc.
- [ ] In the new issue, delete the section above `Dependency` based on the issue creation instructions.
- [ ] Choose `Save` to update the new issue you just revised. 
- [ ] Once all the customization is done, glance it over to see if everything makes sense and there is no more `[Replace ]` in the new issue.
- [ ] Leave the issue in the `New Issue Approval` column.
- [ ] Notify the Research lead or the Product Manager.
- [ ] Research lead reviews the new issues set up
   - Adding the `ready for prioritization label` to each as approved
- [ ] When all issues are correct, the research lead adds the following at the top of the issue
   ```
   ### Dependency
   - [ ] All the sub issues are closed
   ```

####  
- [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
- [ ] Review with UX Lead
- [ ] Product sign-off
    

### Resources/Instructions
#### Resources for creating this issue
1.01 [RP[Replace 000] issues on GitHub](https://github.com/hackforla/internship/issues?q=state%3Aopen%20label%3A%22Research%3A%20RP[Replace 000]%22)
1.02 [Create Easy Retro Board Template]
1.03 [Consolidate Easy Retro Board Template]
1.04 [1password](https://start.1password.com/signin?l=en)
1.05 [Easy Retro](https://easyretro.io/)

**Comment by gayathri3120** [comment]
Update: Customized the Create and Consolidation Templates to include the "TYPE OF BOARD" to be created. 
Created an Epic Issue template for each Research Plans

As of UX Team meeting on Feb 10th, 
Need to create 3 new accounts, naming them in format "Easy Retro for "Replace TYPE OF BOARD" Interviews
Add examples to the Replace step (ex: "Training," "Practice,", or "Real-time")
Customize the Easy Retro account names in 1Password and add them to the templates to signify which account to use for each particular issue
Epic Issue can be made with simple instructions, remove the Customize Resource links step, add the step to add subissues and relationships (use #427, #725, #782, #326 for examples)

**Comment by gayathri3120** [comment]
Update as of UX Team meeting on Feb 17th,
Need to add the account instructions to the Consolidation Template
Question: After consolidation, should the assignee delete the board or keep on creating new accounts for new RP?

**Comment by gayathri3120** [comment]
Update: Done editing all 3 templates
Blocker: This is part of the Create template is not showing up properly @pandanista 

<img width="1152" height="385" alt="Image" src="https://github.com/user-attachments/assets/a11a9736-566f-4e3c-8a00-e3b18f86aa31" />

I want the text to appear after the "Copy". I pasted it there while editing but after clicking Update, it still shows up on top.

**Comment by pandanista** [comment]
> Update: Done editing all 3 templates Blocker: This is part of the Create template is not showing up properly [@pandanista](https://github.com/pandanista)
> 
> <img alt="Image" width="1152" height="385" src="https://private-user-images.githubusercontent.com/76033038/551312460-a11a9736-566f-4e3c-8a00-e3b18f86aa31.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzMyNDg2NDcsIm5iZiI6MTc3MzI0ODM0NywicGF0aCI6Ii83NjAzMzAzOC81NTEzMTI0NjAtYTExYTk3MzYtNTY2Zi00ZTNjLThhMDAtZTNiMThmODZhYTMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAzMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMzExVDE2NTkwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3ZDQzYzlhZmI5Y2Q0YmY5ZGUwYWIxMWVlNGJlYWQ2N2QyMDJjMGYxZDNlZGMyNTdiYTZhNWM5MDExYTA3MGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.vdkBn9sSusqa5LIq_9DMGWhasdp1TPTPnerWEmAqR5s">
> I want the text to appear after the "Copy". I pasted it there while editing but after clicking Update, it still shows up on top.

- Thank you for editing the templates. I wouldn't worry about the entire text showing up in the section you mentioned above. Since the goal is for assignee to copy and paste the title you drafted, and the copy button serves the purpose, so they look good to me. 
- Adding 'ready for product' label so Product can review and answer questions.
